src/finch/autoschedule/compiler.py

```diff
@@ -1,133 +1,81 @@
-from textwrap import dedent
-from typing import TypeVar, overload
+import numpy as np
 
-from ..algebra import fill_value
+from .. import finch_notation as ntn
+from ..algebra.tensor import TensorFormat
 from ..finch_logic import (
+    Aggregate,
     Alias,
     Field,
     Literal,
     LogicExpression,
     LogicNode,
-    LogicTree,
     MapJoin,
+    Plan,
+    Produces,
     Query,
     Reformat,
     Relabel,
     Reorder,
-    Subquery,
     Table,
     Value,
 )
-from ._utils import intersect, with_subsequence
-
-T = TypeVar("T", bound="LogicNode")
-
-
-@overload
-def compute_structure(
```

Comment:
I removed `compute_structure` stubs&function from here. I added them some time ago in the initial draft but it's not used.

Reply:
we need the function eventually for caching compiled kernels.

Reply:
We need compute_structure to eventually normalize the program so we can hash it and cache the compiled kernel.

Reply:
Sure! Reverted.